datastore/db/queries/picture/__init__.py

```diff
@@ -17,42 +17,65 @@ class PictureSetNotFoundError(Exception):
 class PictureUpdateError(Exception):
     pass
 
+class GetPictureSetError(Exception):
+    pass
 
 """
 This module contains all the queries related to the Picture and PictureSet tables.
 """
 
-
-def new_picture_set(cursor, picture_set, user_id: str):
+def new_picture_set(cursor, picture_set, user_id: str, folder_name: str = None):
```

Comment:
I assume a picture_set still has a name on the tool even if it isn't named properly (ence why we make it optional). However I think we should still save a name if there is no name given to represent the default given name of the folder? WDYT?